### ISSUE TITLE
Is there a way to hide the panel on a story basis rather than in the global settings?

### ISSUE THREAD

**Issue by tkdphoenix** [storybookjs/storybook#14442]
**Is your feature request related to a problem? Please describe**
My use case includes @storybook/web-components. I presume it would not be different for other frameworks. I have a scenario where I want to display available icons, but there are no actions required. In versions prior to 6.2.1 I could write something like `options: { showPanel: false } }` and the bottom panel would not show up. I see this page for global settings: https://storybook.js.org/docs/react/configure/features-and-behavior but nothing that explains how to do this on a story basis. 

**Describe the solution you'd like**
I would like to understand where this can be set in version 6 and if it is not available to have it as a feature request. If it is available, to have it explained to me, and I can contribute to the documentation.

This is similar to this ticket: https://github.com/storybookjs/storybook/issues/6699 yet still different. I want to be able to hide various tabs or the entire lower panel in this case.

**Describe alternatives you've considered**
Here is an example of my code today:
```import { html } from 'lit-element/lit-element'
import { AmexIcons } from '../src/amex-icons'

export default {
  title: 'Components/amex-icons',
  component: AmexIcons,
}

const Template = () => {
  html'<really long template of icons to display' // these are backticks in my code 
}

export const AllIcons = Template.bind({})
```

I presume there is a method I can use with `AllIcons` but I don't see anything in documentation to describe something like:

```
AllIcons.parameters = {
  showPanel: false,
}
```

Can you guide me to be able to hide the lower panel for this story?

**Are you able to assist to bring the feature to reality?**
Yes, I can write documentation, but I haven't worked with Storybook before and wouldn't be able to write the code, if the feature doesn't exist.

**Additional context**

**Comment by shilman** [comment]
I don't believe this is configurable on a per-story basis anymore. @ndelangen?

**Comment by marcellino-ornelas** [comment]
I had a similar use case where I wanted to show the addon panel to all my stories but wanted to disabled it in a specific story that I didn't need it.

Adding a `showPanel` option to my specific story seemed to apply to all my stories. Not sure why 🤷‍♂️ 

I was able to solve this by adding a global `showPanel` option to `true` in `preview.js` and then overriding that option in each story that I didn't want to show 

`preview.js`
```ts
export const parameters = {
	options: {
		showPanel: true,
	},
	/* ...rest */
};
```

Then in my story file I was able to hid the panel with:

```ts
export default {
	title: 'Button',
	component: Button,
	parameters: { options: { showPanel: false } },
} as ComponentMeta<typeo f Button>;
```

This would hide the panel for all stories in that file. If you want a specific story turned off you can use

```ts
export const Primary = Template.bind({});
Primary.parameters = {
	options: { showPanel: false },
};
```

**Comment by shilman** [comment]
We’re cleaning house! Storybook has changed a lot since this issue was created and we don’t know if it’s still valid. Please open a new issue referencing this one if:
- this is still a problem in SB 7.x
- you can provide a consistent reproduction in 7.x (at https://storybook.new/ or per [repro docs](https://storybook.js.org/docs/react/contribute/how-to-reproduce)).

**Comment by evgeniy-arapov** [comment]
Maybe it will be useful to someone.

If we define     `options: {showPanel: true}` in `preview.ts` then `showPanel: false` for `story` turns off the panel only for story.

**Comment by Smeagolworms4** [comment]
I search a solution for hide story in panel and adddonDocs

**Comment by neerajabandi** [comment]
I added  export const parameters = {
    controls: { disable: true },
  }; in preview.js file and in my story file I added  parameters: {
    controls: { disable: false }, } but its not working, I am using storybook 7

**Comment by audunolsen** [comment]
Please provide us with an easy way to disable the addons-panel on a per story basis. If this is possible then make it obvious how to do so in the documentation. I can't find a way to achieve this in SB > 8.

We want to disable the whole panel that is toggled in the SB ui with the A key, not individual tabs inside it.

**Comment by miladfm** [comment]
any update?

**Comment by korjikk** [comment]
This would be a handy option.

**Comment by eliasmohsengrenvald** [comment]
We use Storybook v8 and would love to have a possibility to disabled addons-panel per story too and @marcellino-ornelas solution didn't work for us.
I tried manually adding `panel=false` query parameter and that indeed hides the addons-panel, however, I'm not sure how to achieve that programatically in Storybook 8

**Comment by conceptree** [comment]
I am also creating a new storybook project with v8 and struggling a bit. In my case, i want to disable the top action bar wheren the theme toggle lives, in all of my mdx doc files. For the jsx stories of my web components I need the panel but not for the docs.

I tried 

<Meta title="Introduction" parameters={{ actions: { disable: true } }} />

and 

<Meta title="Introduction" parameters={{ options: { showPanel: false} }} />

Any ideas?

Thanks in advance

**Comment by TomDeSmet** [comment]
This worked for me for controls and actions (not for interactions):

```
const meta = {
  parameters: {
    controls: {
      disable: true,
    },
    actions: {
      disable: true,
    },
  },
} satisfies Meta;
```

**Comment by mariohernandez** [comment]
The solution from @TomDeSmet works for hiding the controls and actions panels, but the interactions panel is still visible and can't hide it. Any ideas?
My last option is using CSS to hide the panel, but would like to avoid that if possible.

**Comment by averagecupofjoey** [comment]
Has anyone found a solution that works for this? I've spent **hours** looking at workarounds posted here and the docs and nothing is working for me. 

In a single story if I set:

```
   parameters: {
    layout: 'fullscreen',
    options: {
      bottomPanelHeight: 0
    }
```

It hides the panel for every story. If I set it to another height in a different story, it only takes the height of the first story you go into, and doesn't apply it to the others. I've also tried setting a global height in the preview.ts file and that didn't fix it. 

I'm trying to have a section where it's simply documentation that doesn't show the addons panel, but it seems to be either EVERY story will show it by default, or EVERY story will hide it. I'm using Angular 19 and Story book 8.6 if that's important.

Thanks for the help!

**Comment by nathaliafleite** [comment]
Any update on this? We use Storybbok v8+ and also really need this feature for some of our stories.
